AppCenterReactNativeShared/android/build.gradle

```diff
@@ -27,7 +27,7 @@ allprojects {
 
     android {
         compileSdkVersion 23
-        buildToolsVersion '23.0.1'
+        buildToolsVersion '27.0.3'
```

Comment:
We must not hardcode a version as it could break in existing projects. It should be managed by linker script dynamically.